src/api/gitHub.js

```diff
@@ -31,72 +32,27 @@ let count = 0;
 /**
  * Org metrics
  */
-const orgMetrics = {
+let orgMetrics = {
   mostPr: 0,
   mostIssues: 0,
 };
 
+/**
+ * https.Agent, only used if the selfsigned option was passed
+ */
+let agent = {};
+
 /**
  * Fetch 100 repositories at a cursor given Organization and valid PAT
  *
  * @param {string} org the organization
  * @param {string} token the token
  * @param {string} cursor the last repository fetched
+ * @param {boolean} selfsigned the selfsigned option 
  * @param {[Objects]} server the graphql endpoint for a GHES instance
  */
-export const fetchRepoInOrg = async (org, token, server, cursor) => {
-  return await fetch(await determineGraphQLEndpoint(server), {
-    method: "POST",
-    headers: {
-      Authorization: `bearer ${token}`,
-    },
-    body: JSON.stringify({
-      query: `{
-        rateLimit {
-          limit
-          cost
-          remaining
-          resetAt
-        }
-        organization(login: "${org}") {
-          repositories(first: 50${cursor}){
-            totalCount
-            edges {
-              cursor
-              node {
-                projects(first:1){
-                  totalCount
-                }  
-                hasWikiEnabled
-                issues(first: 1) {
-                  totalCount
-                }
-                pullRequests(first: 1) {
-                  totalCount
-                }
-                discussions(first: 1) {
-                  totalCount
-                }
-                packages(first: 1) {
-                  totalCount
-                }
-                releases(first: 1) {
-                  totalCount
-                }
-                name
-                id
-                url
-                pushedAt
-                isPrivate
-                isArchived
-                diskUsage
-              }
-            }
-          }
-        }
-      }`,
-    }),
-  })
+export const fetchRepoInOrg = async (org, token, server, selfsigned, cursor) => {
+  return await fetch(await determineGraphQLEndpoint(server), await fetchRepoInOrgInfoOptions(org, token, cursor, selfsigned))
```

Comment:
The two parameters don't need to use `await`:

```suggestion
  return await fetch(determineGraphQLEndpoint(server), fetchRepoInOrgInfoOptions(org, token, cursor, selfsigned))
```

Reply:
I changed the logic as suggested.